tests/flow-force-reuse/flow-force-reuse-off/test.yaml

```diff
@@ -0,0 +1,16 @@
+pcap: ../test.pcap
+
+args:
+- --set flow.memcap-policy=drop-flow
+- --set flow.prealloc=0
+- --set flow.memcap=3kb
+- --set flow.hash-size=32
+- --set flow.force-reuse=false
+
+checks:
+  - shell:
+      args: grep 'flow.memcap ' stats.log | wc -l
```

Comment:
prefer eve.json stats checks here

Reply:
The number of counters on flow.memcap and flow.get_used isn't always the same, so the grep is just checking if there is any. Is this comment to use the match syntax of test.yaml or continue using grep but grep eve.json instead?

Thanks